### PR TITLE
Poke template emojis

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR is final step towards supporting emojis on template. It fixes https://github.com/dust-tt/tasks/issues/675.

This update enhances the template form on Poke by incorporating pickers instead of relying on plain text values. Additionally, it modifies the assistant builder path to rely on a generated pictureUrl in the same way a regular assistant does. 

Instead of updating the table for template, this PR hacks the way we store emoji so we can easily build a valid visual url from it.

templateEmoji

BuilderWithEmoji

## Risk

Until we update the emojis in Poke the template visual will look pretty broken in the UI (some text overlapping the color -- should last for 1 minute).

### CODE REVIEW

File: front/components/assistant_builder/AssistantTemplateModal.tsx [hunk 1]
```diff
@@ -60,12 +54,7 @@ export function AssistantTemplateModal({
       <Page variant="modal">
         <div className="flex flex-col gap-5 pb-6">
           <div className="flex max-h-32 max-w-lg flex-row gap-3">
-            <Avatar
```
Comment (philipperolet) [inline front/components/assistant_builder/AssistantTemplateModal.tsx:63]: :+1: 
Thinking out loud: would it make sense to remove emoji/bgcolor as props of avatar, and always use urls throughout our code?

File: front/lib/resources/template_resource.ts [hunk 2]
```diff
@@ -108,12 +109,24 @@ export class TemplateResource extends BaseResource<TemplateModel> {
     return this.visibility === "published";
   }
 
+  get pictureUrl() {
```
Comment (philipperolet) [inline front/lib/resources/template_resource.ts:112]: :open_mouth: what does `get` do? :student:

**Review by philipperolet — APPROVED** [review]
:+1: (coms are just discussions, not suggesing changes)

File: front/lib/resources/template_resource.ts (hunk 2 above)
Comment (flvndvd) [reply front/lib/resources/template_resource.ts:112]: It's a getter: https://www.typescriptlang.org/docs/handbook/2/classes.html#getters--setters.

File: front/components/assistant_builder/AssistantTemplateModal.tsx (hunk 1 above)
Comment (flvndvd) [reply front/components/assistant_builder/AssistantTemplateModal.tsx:63]: There are still some places where it's being used through emoji and background color.

**Pull request merged**